internal/testrunner/runners/system/runner.go

```diff
@@ -174,6 +176,10 @@ func (r *runner) run() (results []testrunner.TestResult, err error) {
 	return results, nil
 }
 
+func createTestRunID() string {
+	return fmt.Sprintf("%d", rand.Intn(99999-10000)+10000)
```

Comment:
Maybe seed the PRNG each time with something like `rand.Seed(time.Now().UnixNano())`?

Also, nit: for an extra bit of readability, consider defining min and max constants and then using those in the `rand.Intn` call.

Reply:
Fixed